services/agent/tests/conftest.py

```diff
@@ -182,14 +182,14 @@ def caplog_info_debug(caplog: LogCaptureFixture) -> Iterable[LogCaptureFixture]:
         yield caplog
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
 def mocked_s3_server_url() -> Iterator[HttpUrl]:
```

Comment:
Actually, check ``pytest_simcore.aws_services`` since there is already a s3 server fixture in there. YOu might want to reuse it.

https://github.com/ITISFoundation/osparc-simcore/blob/59d92328b0b0062112743bfff99dfeaadb0aa492/packages/pytest-simcore/src/pytest_simcore/aws_services.py

Reply:
thanks using this is better!